components/dashboard/src/settings/Plans.tsx

```diff
@@ -294,8 +339,8 @@ export default function () {
     } else {
         const targetPlan = freePlan;
         let bottomLabel;
-        if (scheduledDowngradePlanId === targetPlan.chargebeeId) {
-            bottomLabel = <p className="text-green-600">Downgrade scheduled<br/><a className="text-blue-light leading-6" href="javascript:void(0)">Cancel</a></p>;
+        if (scheduledDowngradePlanId === targetPlan.chargebeeId && !pendingDowngradeCancellation) {
+            bottomLabel = <p className="text-green-600">Downgrade scheduled<br/><a className="text-blue-light leading-6" href="javascript:void(0)" onClick={() => confirmCancelDowngrade()}>Cancel</a></p>;
```

Comment:
**issue**: Similarly, when upgrading we are adding the following copy.

```
Upgrade in progress
```

Should we align these two like the following?

```
Upgrading ...
Downgrading ...

or

Upgrade in Progress ...
Downgrade in Progress ...
```

Reply:
This is also not relevant anymore, see https://github.com/gitpod-io/gitpod/pull/3873#discussion_r615992156.